packages/bindings-test/src/multitest.rs

```diff
@@ -153,6 +153,29 @@ impl Pool {
         }
     }
 
+    // TODO: implement twap logic
```

Comment:
For existing gamm bindings queries, defining pool methods was possible because they only require the state of the pool (e.g https://github.com/osmosis-labs/bindings/blob/main/packages/bindings-test/src/multitest.rs#L69-L128) , this is not possible with implementing a method for twap. Does reviewers have any recommendation for this situation?

Reply:
A possible plan rn that I'm thinking of is adding a field in https://github.com/osmosis-labs/bindings/blob/main/packages/bindings-test/src/multitest.rs#L28 for timestamp for twap

Reply:
I think we should just not implement the mocks, mocks are so broken that I'd rather discourage anyone from ever using them

Or if we have to implement the mock, just return spot price for now or an error

Reply:
Changed to return spot price for now